### PR TITLE
Logic

### PR DESCRIPTION
**Pull request by misterpaul4**
Created the main logic for where you have a specific winner and a draw if it was given.

### CODE REVIEW

**Comment by pelzolga123** [comment]
## Hi Team!  
### Your project looks :+1: great!
Still, there are some changes needed before this Pull Request can be approved:
* [ ] Pull Request should have short description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
* [ ] No `puts/print` and `get.chomp` outside `main.rb` file. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/only_main_print.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))
* [ ] The game returns an error `cannot load such file -- colorize` when trying to run it. To fix said error add a Gemfile with `gem 'colorize` inside.

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

File: bin/main
Comment (pelzolga123) [inline bin/main]: [OPTIONAL] Try to avoid using global variables in your project. It's considered a bad practice, you can read more about it [here](https://www.tutorialspoint.com/ruby/ruby_variables.htm).

**Review by MarioDena — CHANGES_REQUESTED** [review]
Hello! Good job implementing the requested changes! 👍 

However you have some small fixes pending before approval.

## Required changes

- [ ] Please remember to delete any unrelated files from your project the .gitkeep file on your lib is no longer used/necessary 

## Optional Changes

- [ ] I would not use global variables since it's considered a bad practice and will not give you approval of the end project/final milestone.

After making the requested changes please resubmit your project for review.

**Comment by uryelah** [comment]
Hello **Fernando Rivas Medina** & **Paul Ajuizeogu**, I'm Sarah, your last reviewer for this PR 👩‍🚀  

---------    
    
Good job fixing the last required change.

---------

## **Your project is approved! 🎊 🥅 💛**

**You may close this Pull Request and merge the changes**


---------

    
    
**Keep being awesome devs and good luck with your next project!**

---------

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Pull request merged**
